src/Neighbor/NodePairIdxType.hh

```diff
@@ -1,12 +1,11 @@
-#ifndef _Spheral_NodePairIdxType_hh_
-#define _Spheral_NodePairIdxType_hh_
+#ifndef Spheral_NodePairIdxType_hh
+#define Spheral_NodePairIdxType_hh
 
 #include "Utilities/size_t_bits.hh"
 #include "Utilities/DBC.hh"
 
 #include <iostream>
-#include <iostream>
-#include <functional>   // hash
+#include "config.hh"
```

Comment:
This is nitpicky, but I try to keep the includes sorted by origin (std C++ in one block, our local headers in another).  I think "config.hh" is one of our -- could it by moved up with the other local Spheral source includes?